src/main/kotlin/lotto/domain/Lotto.kt

```diff
@@ -12,13 +12,13 @@ class Lotto(
         require(numbers.size == LOTTO_SIZE) { "Lotto must contain exactly 6 numbers." }
     }
 
+    constructor(vararg numbers: Int) : this(numbers.map(LottoNumber.Companion::from))
+
```

Comment:
I see you already appreciate the order of class layout 👍

https://kotlinlang.org/docs/coding-conventions.html#class-layout